tests/dragonfly/server_family_test.py

```diff
@@ -172,3 +173,38 @@ async def test_blocking_commands_should_not_show_up_in_slow_log(
 
     # blpop does not show up, only the previous reset
     assert reply[0]["command"] == "SLOWLOG RESET"
+
+
+@dfly_args({"memcached_port": 11211})
+async def test_metric_labels(
+    df_server: DflyInstance, async_client: aioredis.Redis, memcached_client: Client
+):
```

Comment:
We do not need a test for the prometheous metrics but I guess it won't hurt ? You can just open your browser and check via `http://localhost:6379/metrics` or just rely on the unit test. Not strongly opinionated about this tbh

Reply:
Also you don't need `memcached_client` or `memcached_port`

Reply:
I used memcached client to get a count of `other` connection and command as I see it goes via a listener other than main. Do you recommend to remove this test?

Reply:
it's actually an interesting point. We want to count the memcache listener as "main" as well. it is customer facing as well.

Reply:
I added a new method which returns true if the role is `MAIN` _or_ the protocol is `MEMCACHED` and adjusted the metrics to use this. The test is also changed to account for this and use the admin port for counting `other` metrics.

This will metric will be incremented for any memcached listener (or main role listener),  I hope this is correct behavior.